app/src/main/res/layout/fragment_profile_step_two.xml

```diff
@@ -91,7 +90,7 @@
           android:layout_marginTop="@dimen/padding_medium_small"
           android:gravity="center_horizontal"
           android:text="@string/create_profile_pub_pri"
-          style="@style/Aptoide.TextView.Medium.L.Theme"
+          style="@style/Aptoide.TextView.Medium.L"
```

Comment:
same as before, this style is not defining any color for it is this intentional?

Reply:
solved